src/data/complex/exponential.lean

```diff
@@ -355,6 +355,15 @@ the complex exponential function -/
 /-- The complex tangent function, defined as `sin z / cos z` -/
 @[pp_nodot] def tan (z : ℂ) : ℂ := sin z / cos z
 
+/-- The complex secant function, defined as `1 / cos z` -/
```

Comment:
I don't think these are used in this PR, right? they are useful but i'd put them somewhere else